blobfuse2-code-coverage.yaml

```diff
@@ -62,33 +62,6 @@ stages:
               value: '$(System.DefaultWorkingDirectory)/azure-storage-fuse'
 
           steps:
-            - checkout: none
```

Comment:
I suggest we skip this part for now and take is as a change after the release. lets not fix too many things and break more.